scale_out/test/RingMultiCgraRingCtrlMemRTL_test.py

```diff
@@ -81,6 +81,7 @@ def done(s):
     # return True
 
   def line_trace(s):
+    # return s.dut.verbose_trace(verbosity = 2)
```

Comment:
If this is the way to invoke the verbose_trace(), let's enable it by default for this test?

Reply:
verbose_trace works for `-xvs --tb=short` but not `--test-verilog`, it looks up `line_trace()`, can test this with `-xvs --tb=short` for RingMultiCgraRingCtrlMemRTL_test.py, current is [`pytest --tb=short -sv ../scale_out/translate/RingMultiCgraRingCtrlMemRTL_test.py --test-verilog --dump-vtb --dump-vcd`](https://github.com/tancheng/VectorCGRA/blob/49a40187f90a8d67b0a410c0d3bc5944422e22e6/.github/workflows/python-package.yml#L82)?

Reply:
- Then the `xxx_test.py` within `translate/` folder can have `line_trace()`, while the one in `test/` can have vervose_trace() enabled by default.

 - Or I guess actually you can control it based on the flag?

```
  def line_trace(s, verbosity=0):
    if verbosity = 0:
      return s.dut.line_trace(verbosity)
    if verbosity = 1:
      return s.dut.verbose_trace(verbosity)

......

dut = Top()

if flags not contains --test-verilog:
  dut.set_param('top.x.y.z.line_trace', verbosity=1) # Or use a global variable?
```